serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -40,8 +63,6 @@ class SerializabilityCheckerCompilerPluginComponent(
         ("akka.actor.typed.ActorRef.tell", Seq(ClassType.Message)),
         ("akka.actor.typed.RecipientRef.tell", Seq(ClassType.Message)))
 
-      private val ignoredTypePrefixes = List("akka.", "scala.Any", "scala.Nothing")
```

Comment:
Actually... how is the `akka.` prefix handled now then?

Reply:
I moved this `val` up the file, to line 44